### PR TITLE
Use aws-iam-role terraform module to create IAM Role

### PR DESCRIPTION
**Pull request by oktaviandi-nugraha**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -25,23 +38,13 @@ resource "aws_iam_role_policy" "lambda_additional_policy" {
...
-    ManagedBy     = "Terraform"
-  }
+  prefix      = "${var.service_name == "" ? var.product_domain : var.service_name}-${var.lambda_name}-"
 }
 
 resource "aws_lambda_function" "lambda_classic" {
   s3_bucket     = "${var.lambda_code_bucket}"
   s3_key        = "${var.lambda_code_path}"
-  function_name = "${random_id.randomiser.hex}"
+  function_name = "${local.full_name}"
```
Comment (arief-halim) [inline main.tf:47]: Shouldn't we use https://github.com/traveloka/terraform-aws-resource-naming for lambda function name?

**Comment by sshvetsov** [comment]
As I have already mentioned in the previous pull request, I do not believe this module makes sense. Since there has been no response/discussion I prefer to close this pull request and delete the module entirely.

**Comment by oktaviandi-nugraha** [comment]
@sshvetsov what do you think the better approach then? Create a new module for Lambda? The periodic worker then could use the lambda module and create aws cloudwatch event? 

I think IAM role creation should stay in @rafikurnia module so we could ensure IAM role creation is consistent. The lambda module needs to call this module to create IAM role and attach additional permission.

File: main.tf (hunk 1 above)
Comment (oktaviandi-nugraha) [reply main.tf:47]: Will do

**Pull request closed without merging**